File: C Modules/Assignments/Assignment no 7/Question-2/Assignment no 3/Type-3/Question_10.c
#include<stdio.h>
int sum()
{
    int n, first, last;
    int *ptr1=&n;
    int *ptr2=&first;
    int *ptr3=&last;
    printf("Enter the number:\n");
    scanf("%d", ptr1);

    *ptr3 = (*ptr1) % 10;   
    while (*ptr1 >= 10)
    {
        *ptr1 = *ptr1 / 10;   
    }
    *ptr2 = *ptr1;      
    return *ptr2+*ptr3;
}

int main()
{
    int result=sum();
    int *ptr=&result;
    printf("Sum :%d",*ptr);
    return 0;
}

File: Java Module/Assignments in c to java/Assignment-03/Question_10.java
public class Question_10 {

    public static void main(String[] args) {
        {
    int n=10, first, last;

    // printf("Enter the number:\n");
    // scanf("%d", &n);

    last = n % 10;   
    while (n >= 10)
    {
        n = n / 10;   
    }
    first = n;      
    System.out.printf("Sum :%d", first + last);
    }
}
}